stdlib/Manifest.toml

```diff
@@ -67,7 +67,7 @@ version = "1.11.0"
 
 [[deps.JuliaSyntaxHighlighting]]
 deps = ["StyledStrings"]
-uuid = "dc6e5ff7-fb65-4e79-a425-ec3bc9c03011"
+uuid = "ac6e5ff7-fb65-4e79-a425-ec3bc9c03011"
```

Comment:
I'm a little confused, it seems that the UUID of `JuliaSyntaxHighlighting.jl` is specified as `dc6e5ff7-fb65-4e79-a425-ec3bc9c03011` from the first commit.
So the current UUID seems to be correct?

https://github.com/JuliaLang/JuliaSyntaxHighlighting.jl/blame/19bd57b89c648592155156049addf67e0638eab1/Project.toml#L2

Reply:
Yea, so the mistake at the heart of this is:

> development of the backported version of this stdlib, to avoid potential sysimage issues the UUID was temporarilly changed and I forgot to change it back.

The changed UUID (`ac6e...`) is what was registered in General as the backport/compat version: https://github.com/JuliaRegistries/General/blob/master/J/JuliaSyntaxHighlighting/Package.toml#L2